File: Programación y Algoritmos/Actividad 13/temperatura.cpp
#include <iostream>
#include <string>
#include <iomanip>

using namespace std;

float fahrenheit;
float celsius;
string mensaje;

int main() 
{
    cout << "Introduzca la temperatura en grados Fahrenheit" << endl;
    cin >> fahrenheit;
    celsius = (fahrenheit - 32) * 5/9;
    cout << "La temperatura dada equivale a " << setprecision(4) << celsius << " grados Celsius" << endl;

    if (celsius >= 20 && celsius <= 25)
    {
        mensaje = "ideal";
    }
    else if (celsius < 20)
    {
        mensaje = "no clasificada";
    }
    else if (celsius > 25 && celsius <= 32)
    {
        mensaje = "peligrosa";
    }
    else if (celsius > 32)
    {
        mensaje = "contraindicada";
    }

    cout << "Esta temperatura es " << mensaje << endl;
    

}

File: Programación y Algoritmos/Actividad 13/temperatura.py
fahrenheit = float(input("Introduzca la temperatura en grados Fahrenheit: "))

celsius = (fahrenheit - 32) * 5/9

if celsius < 20:
    mensaje = "no calsificada"
    
elif celsius > 20 and celsius <= 25:
    mensaje = "ideal"

elif celsius > 25 and celsius <= 32:
    mensaje = "peligrosa"
elif celsius > 32:
    mensaje = "contraindicada"

print(f"La temperatura equivale a {celsius:.2f} grados Celsius, por lo que es {mensaje}")